qemu/tests/tpm_bind_luks.py

```diff
@@ -53,7 +53,11 @@ def get_md5sum():
         """
         Return the file's MD5
         """
-        return session.cmd_output("md5sum " + dd_file).split()[0]
+        status, md5 = session.cmd_status_output(
+            "md5sum " + dd_file, print_func=test.log.info).split()[0]
```

Comment:
Got error: ERROR: 'tuple' object has no attribute 'split'. No need split()[0].

Reply:
Thank you. Forgot to update this line, corrected it.